documentation/project-docs/SDK-PR-guide.md

```diff
@@ -0,0 +1,50 @@
+# SDK PR Guide
+
+## Branching
+See the [versions](https://learn.microsoft.com/en-us/dotnet/core/porting/versioning-sdk-msbuild-vs#lifecycle) document for more details on how SDK versions map to MSBuild and Visual Studio.
+
+### Major releases
+The .NET SDK has yearly major releases aligning with the .NET Runtime in November.  These releases will be of the form N.0.1xx. The feature work for major releases will be done in `main`. 
+Typically, previews for each major release start in February of each year.  For each preview, we will snap a new branch off of main approximately midway through the prior month.
+For .NET 7 for example, the preview branches were created typically around the 21st of each month prior with the later previews forking earlier to provide more time for stabilization and codeflow.
+Main branch has moved to be the next year's release branch in August.
+
+### Minor releases
+The .NET SDK has quarterly minor releases that align with Visual Studio releases. These SDK releases are open to all bug fixes and some features work as well. 
+These quarterly releases should avoid breaking changes if at all possible. Breaking changes instead should be included in the major releases and be tied to customer TargetFramework updates when possible.
+
+Minor releases lock down after preview 3 for each release.  Preview 3 is typically 1 month prior to GA and locks down by the end of the month before the preview release.
+
+### Servicing releases
+The .NET SDK has monthly servicing releases aligning with the .NET runtime servicing releases. These are for top fixes and security updates only to limit risk.
+Any servicing release is open for checkins from the day the branding PRs are merged (~1st of each month) and when code complete is (typically two weeks later).
+The servicing branches are locked from the time of code complete to the next branding in case we need to respin any monthly release. Final signoffs are typically in the last week of each month.
```

Comment:
I assume we branch out or create a tag after code complete. For example, I see a tag created for December 2022 servicing https://github.com/dotnet/sdk/tree/v7.0.101.

I assume tags are created at the time of code complete. How about opening servicing branch for code flow without waiting for next branding PRs to be merged?

Reply:
I think it's to prevent confusion where people merge, a build from the prior month gets created, and then someone tests it thinking it was last months build. I don't think there are specific engineering reasons other than avoiding confusion. @mmitche to confirm?